### PR TITLE
[4.0] Namespace mod articles latest

### PR DESCRIPTION
**Pull request by C-Lodder**
### Summary of Changes

Namespace mod articles latest

### CODE REVIEW

File: modules/mod_articles_latest/Helper/ArticlesLatestHelper.php [hunk 1]
```diff
@@ -7,21 +7,24 @@
  * @license     GNU General Public License version 2 or later; see LICENSE.txt
  */
 
-defined('_JEXEC') or die;
+namespace Joomla\Module\ArticlesLatest\Site\Helper;
 
-JLoader::register('ContentHelperRoute', JPATH_SITE . '/components/com_content/helpers/route.php');
+defined('_JEXEC') or die;
```
Comment (brianteeman) [inline modules/mod_articles_latest/Helper/ArticlesLatestHelper.php:12]: shouldnt this be above the namespace line

Comment (mbabker) [reply modules/mod_articles_latest/Helper/ArticlesLatestHelper.php:12]: A namespace declaration has to be the first keyword in a PHP file (except for if using a `declare()` keyword, then that is first).

Comment (brianteeman) [reply modules/mod_articles_latest/Helper/ArticlesLatestHelper.php:12]: ok - still learning this

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on f1c6f4383b1c81d5870c86e717bf70c1c72e83a8<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17287">issues.joomla.org/tracker/joomla-cms/17287</a>.</sub>

**Pull request merged**